sdk/python/kfp/components/executor.py

```diff
@@ -98,8 +101,64 @@ def _get_input_artifact_path(self, artifact_name: str):
               artifact_name))
     return input_artifact.path
 
-  def _write_executor_output(self):
-    executor_output = {'artifacts': {}}
+  def _write_output_parameter_value(self, name: str, value: Union[str, int,
+                                                                  float]):
+    if type(value) == str:
+      output = {"stringValue": value}
+    elif type(value) == int:
+      output = {"intValue": value}
```

Comment:
Are integers planned to be supported in MLMD?

Reply:
This is how it's written in ExecutorOutput.

Reply:
JFYI: I've heard there are some plans to change ExecutorOutput.